zerver/openapi/zulip.yaml

```
@@ -1860,6 +1860,10 @@ paths:
                                     has to be muted, the second element is the topic name to be muted
                                     and the third element is an integer UNIX timestamp representing
                                     when the topic was muted.
+
+                                    **Changes**: In Zulip 6.0 (feature level 133), this is
+                                    now returned only when `user_topic` is not specified as
+                                    an `event_type` while registering the event queue.
```

```suggestion
                                    now returned only when `user_topic` is not specified in
                                    `event_types` while registering the event queue.
```

Alternatively, "as an event type" would be acceptable, though less clear than "in `event_types`".

But "as an `event_type`" is wrong. Backticks (or when rendered, a code font) indicate code. So saying "`event_type`" means an identifier `event_type`. There is no such identifier in our API. There is `event_types`, but that is a completely different identifier: identifiers don't do English grammar, including plurals.